chapters/robots.adoc

```diff
@@ -18,6 +18,22 @@ The <<Referee, referee>> has to force a team to remove a robot from the field if
 ==== Shape
 A robot must fit inside a 0.18 meters wide and 0.15 meters high cylinder at any point in time. Additionally, the top of the robot must adhere to the standard pattern size and surface constraints.
 
+==== Robot's Hull Color
```

Comment:
Not totally sure, but I think `'` is wrong here. Maybe rewrite it to "Hull Color of Robots".